### ISSUE TITLE
Uncaught TypeError: Cannot read property 'append' of undefined

### ISSUE THREAD

**Issue by phyng** [atom/settings-view#334]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 0.169.0
**System**: linux 3.13.0-43-generic
**Thrown From**: [settings-view](https://github.com/atom/settings-view) package, v0.166.0

### Stack Trace

Uncaught TypeError: Cannot read property 'append' of undefined

```
At /usr/share/atom/resources/app/node_modules/settings-view/lib/themes-panel.js:204

TypeError: Cannot read property 'append' of undefined
  at /usr/share/atom/resources/app/node_modules/settings-view/lib/themes-panel.js:204:36
  at /usr/share/atom/resources/app/node_modules/emissary/lib/emitter.js:155:30
  at Array.forEach (native)
  at PackageManager.module.exports.Emitter.emitSlow (/usr/share/atom/resources/app/node_modules/emissary/lib/emitter.js:154:47)
  at PackageManager.module.exports.Emitter.emit (/usr/share/atom/resources/app/node_modules/emissary/lib/emitter.js:107:30)
  at PackageManager.module.exports.PackageManager.emitPackageEvent (/usr/share/atom/resources/app/node_modules/settings-view/lib/package-manager.js:368:19)
  at /usr/share/atom/resources/app/node_modules/settings-view/lib/package-manager.js:284:19
  at exit (/usr/share/atom/resources/app/node_modules/settings-view/lib/package-manager.js:43:16)
  at BufferedProcess.triggerExitCallback (/usr/share/atom/resources/app/src/buffered-process.js:63:47)
  at /usr/share/atom/resources/app/src/buffered-process.js:70:18
  at Socket.<anonymous> (/usr/share/atom/resources/app/src/buffered-process.js:150:18)
  at Socket.emit (events.js:129:20)
  at Pipe.close (net.js:469:12)
  at Ipc.sendSync (/usr/share/atom/resources/atom/renderer/api/lib/ipc.js:39:29)
  at Object.RemoteMemberFunction [as showSaveDialog] (/usr/share/atom/resources/atom/renderer/api/lib/remote.js:110:29)
  at Atom.module.exports.Atom.showSaveDialogSync (/usr/share/atom/resources/app/src/atom.js:824:21)
  at Pane.module.exports.Pane.saveItemAs (/usr/share/atom/resources/app/src/pane.js:525:26)
  at Pane.module.exports.Pane.saveItem (/usr/share/atom/resources/app/src/pane.js:515:21)
  at Pane.module.exports.Pane.saveActiveItem (/usr/share/atom/resources/app/src/pane.js:501:19)
  at Workspace.module.exports.Workspace.saveActivePaneItemAndReportErrors (/usr/share/atom/resources/app/src/workspace.js:611:44)
  at Workspace.module.exports.Workspace.saveActivePaneItem (/usr/share/atom/resources/app/src/workspace.js:601:19)
  at atom-workspace.atom.commands.add.core:save (/usr/share/atom/resources/app/src/workspace-element.js:295:30)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/usr/share/atom/resources/app/src/command-registry.js:243:29)
  at /usr/share/atom/resources/app/src/command-registry.js:3:61
  at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/usr/share/atom/resources/app/node_modules/atom-keymap/lib/keymap-manager.js:549:16)
  at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/usr/share/atom/resources/app/node_modules/atom-keymap/lib/keymap-manager.js:391:22)
  at HTMLDocument.module.exports.WindowEventHandler.onKeydown (/usr/share/atom/resources/app/src/window-event-handler.js:167:20)

```

### Commands

```
  2x -3:29.3 window:toggle-menu-bar (atom-text-editor.editor)
     -3:03.4 settings-view:change-themes (atom-text-editor.editor)
  2x -2:47.7 core:backspace (atom-text-editor.editor.mini)
     -0:18.2 fuzzy-finder:toggle-buffer-finder (atom-text-editor.editor.is-focused)
     -0:00.0 core:save (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {
    "themes": [
      "atom-dark-ui",
      "atom-dark-syntax"
    ]
  }
}
```

### Installed Packages

```coffee
# User
No installed packages

# Dev
No dev packages
```

/cc @atom/core

**Comment by Moter8** [comment]
```
[Enter steps to reproduce below:]

...
...
```

These things are there so you write in steps. Please do so.

**Comment by kevinsawicki** [comment]
This is a duplicate of #322 and was fixed in the 0.170 Atom release
